tool/site/server_flask/routes.py

```diff
@@ -322,6 +322,21 @@ def post_release():
     return app.make_response(response)
 
 
+@app.route("/dispense/return", methods=["GET"])
+def get_return():
+    if (config.ENVIRONMENT == "prod"):
```

Comment:
i know this should never be hit but wondering if that has been added for any particular reason?

Reply:
being consistent with previous routes, if we start to consider the use-case for a prod environment then our initial assumption was that this should be read only, so all dispense interactions should be unavailable